modules/RTC/BridgeChannel.js

```diff
@@ -391,21 +375,19 @@ export default class BridgeChannel {
                 break;
             }
             case 'SenderSourceConstraints': {
-                if (FeatureFlags.isSourceNameSignalingEnabled()) {
-                    const { sourceName, maxHeight } = obj;
-
-                    if (typeof sourceName === 'string' && typeof maxHeight === 'number') {
-                        // eslint-disable-next-line object-property-newline
-                        logger.info(`SenderSourceConstraints: ${JSON.stringify({ sourceName, maxHeight })}`);
-                        emitter.emit(
-                            RTCEvents.SENDER_VIDEO_CONSTRAINTS_CHANGED, {
-                                sourceName,
-                                maxHeight
-                            }
-                        );
-                    } else {
-                        logger.error(`Invalid SenderSourceConstraints: ${JSON.stringify(obj)}`);
-                    }
+                const { sourceName, maxHeight } = obj;
+
+                if (typeof sourceName === 'string' && typeof maxHeight === 'number') {
+                    // eslint-disable-next-line object-property-newline
+                    logger.info(`SenderSourceConstraints: ${JSON.stringify({ sourceName, maxHeight })}`);
```

Comment:
Since you are building the same object literal down there to emit the event it might be worth it to create a n auxiliary object to log and emit the event with.

Reply:
Well I just removed the check, was trying to keep refactors to a minimum. I don't see any harm in making the changes you requested.